lib/mtif/posts.rb

```diff
@@ -125,20 +125,26 @@ def mtif_value(value)
       value.kind_of?(Time) ? value.strftime(DATE_FORMAT) : value
     end
 
-    def convert_to_native_type(raw_value)
+    def convert_to_native_type(raw_value, key)
+      if MULTILINE_KEYS.include?(key)
+        return raw_value
+      end
+
+      if key == :date && raw_value.match(/^\d{2}\/\d{2}\/\d{4} \d{2}:\d{2}:\d{2} [AP]M/)
```

Comment:
Makes sense. I hadn't looked to see if there are any other keys which should be dates; I also don't know if anyone has extended this format since it was initially documented ~14 years ago. This is what makes me want to refactor key definitions.